app/styles/input.css

```diff
@@ -332,6 +339,15 @@ input[type=range] {
     padding: 0;
     background: transparent;
 }
+input[type=range]:hover {
+    cursor: grab;
```

Comment:
This isn't mentioned in the commit message?

Reply:
My bad, updated the commit message